### DISCUSSION TITLE
Add label to relationship

### DISCUSSION THREAD

**Discussion by howdu** [filamentphp/filament#4383] [Help]
How can I set the label for a 'Media' resource. 

<img width="965" alt="Screenshot 2022-10-06 at 12 36 23" src="https://user-images.githubusercontent.com/533658/194302796-eb1f0aa2-a681-437f-a33c-d973e4054b61.png">

<img width="909" alt="Screenshot 2022-10-06 at 12 39 11" src="https://user-images.githubusercontent.com/533658/194303410-265ba619-2fab-4fbc-af66-351e9c0c0cc2.png">

Trying to create a many-many relationship with
https://github.com/awcodes/filament-curator

**Comment by Z3d0X** [comment]
Please ask in the plugin repo or plugin channel on our discord
